monitor/monitor.go

```diff
@@ -91,16 +91,22 @@ func NewEthMonitor(ethClient *mainchain.EthClient, operator *transactor.Transact
 		log.Fatalln("ledgerContract err", err)
 	}
 
+	verifiedChanges, err := bigcache.NewBigCache(bigcache.DefaultConfig(10 * time.Minute))
```

Comment:
why 10 min?

Reply:
![image](https://user-images.githubusercontent.com/5875315/83718593-3185bd00-a5ea-11ea-9b66-a34814e2c3fd.png)

Reply:
It is from the an example of big cache. It is not very important for us as long as it is larger than vote expiration time